### PR TITLE
Meetar/texture relief

### PR DESCRIPTION
**Pull request by meetar**
implemented bubble-wrap-style dots transition, and refactored the styles a bit to reduce redundancy

### CODE REVIEW

File: themes/terrain-pattern.yaml [hunk 1]
```diff
@@ -307,43 +318,38 @@ styles:
...
-                    // Draw a circle in the middle of the ST space
-                    float circleBorder (vec2 st, float radius) {
-                                    return stroke(radius, circleDF(st-vec2(0.5)));
-                    }
 
-    # dot terrain from Patricio's example
-    # https://mapzen.com/tangram/play/?scene=https://tangrams.github.io/tangram-sandbox/styles/callejas.yaml&lines=116#10.9329/38.3394/-122.3068
 
-    earth-terrain-pattern:
+    terrain-shade:
```
Comment (nvkelso) [inline themes/terrain-pattern.yaml:324]: `terrain-shading`?

File: themes/terrain-pattern.yaml [hunk 2]
```diff
@@ -307,43 +318,38 @@ styles:
...
-                    // subtract .1 to increase contrast a bit
-                    shade = shade*shade*shade * -1. + .3;
+                    // modify brightness and contrast
+                    shade = shade*shade*shade * -2. + .65;
                     // reset normal to prevent standard terrain shading mixing with dots
                     normal = vec3(0,0,1);
 
-    elevation-ramp-earth:
-        mix: colorized-earth
+    earth-terrain-pattern:
```
Comment (nvkelso) [inline themes/terrain-pattern.yaml:344]: `terrain-pattern-earth`?

File: themes/terrain-pattern.yaml [hunk 3]
```diff
@@ -307,43 +318,38 @@ styles:
...
+        base: polygons
+        mix: [elevation-ramp, terrain-shade, colorized-earth]
+
+    landuse-terrain-pattern:
```
Comment (nvkelso) [inline themes/terrain-pattern.yaml:348]: `terrain-pattern-landuse`?

**Comment by nvkelso** [comment]
Connects with https://github.com/tangrams/refill-style/issues/50

**Comment by meetar** [comment]
@nvkelso name suggestions are fine with me, these came in with the merge from sensescape/texture-relief – recommend you coordinate with her re: naming and I'll follow suit 👍

**Pull request merged**